Spigot-Server-Patches/0076-Optimized-Light-Level-Comparisons.patch

```diff
@@ -64,8 +64,57 @@ index d0fa96651..62ec2ad31 100644
                              // CraftBukkit start
                              // world.setTypeUpdate(blockposition1, Blocks.GRASS.getBlockData());
                              org.bukkit.World bworld = world.getWorld();
+diff --git a/src/main/java/net/minecraft/server/BlockIceFrost.java b/src/main/java/net/minecraft/server/BlockIceFrost.java
+index 950863fb..4b04742a 100644
+--- a/src/main/java/net/minecraft/server/BlockIceFrost.java
++++ b/src/main/java/net/minecraft/server/BlockIceFrost.java
+@@ -19,7 +19,7 @@ public class BlockIceFrost extends BlockIce {
+     }
+ 
+     public void b(World world, BlockPosition blockposition, IBlockData iblockdata, Random random) {
+-        if ((random.nextInt(3) == 0 || this.c(world, blockposition) < 4) && world.getLightLevel(blockposition) > 11 - ((Integer) iblockdata.get(BlockIceFrost.a)).intValue() - iblockdata.c()) {
```

Comment:
I'm also not a fan on how this change spans across multiple patches.

Let's avoid changing all the subsequent patches for this...

And I'm thinking maybe do this in a new patch rather than the old one, so if it needs to be dropped on a major MC update, its easier to do so.

This change concerns me for maintainability.